src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderFactoryWithoutTranslatorBagTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Symfony\Component\Translation\Bridge\Loco\Tests;
+
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
+use Symfony\Component\Translation\Bridge\Loco\LocoProviderFactory;
+use Symfony\Component\Translation\Provider\ProviderFactoryInterface;
+
+/**
+ * @group legacy
+ */
+class LocoProviderFactoryWithoutTranslatorBagTest extends LocoProviderFactoryTest
+{
+    use ExpectDeprecationTrait;
```

Comment:
I guess the full test case can be removed?

Reply:
I think it would be nice to ensure the `LocoProviderFactory` still works even without the `TranslatorBag`. WDYT?